app/cli/logs.py

```
@@ -0,0 +1,144 @@
+    atelier = Atelier()
+    flow_id = _resolve_flow_id(atelier, flow_id)
+
+    if follow:
+        _follow_logs(atelier, flow_id, task, show)
+        return
+
+    try:
+        entries = atelier.store.read_logs(flow_id)
+    except FileNotFoundError:
+        console.print(f"[red]unknown flow:[/red] {flow_id}")
+        raise typer.Exit(code=1)
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)